concent_api/core/views.py

```diff
@@ -55,8 +55,22 @@ def send(request, message):
 
 @api_view
 @require_POST
-def receive(_request):
-    return "message sent"
+def receive(request, message):
+    undelivered_message_statuses    = MessageStatus.objects.filter(delivered = False)
+    last_undelivered_message_status = undelivered_message_statuses.order_by('id').last()
```

Comment:
1) `id` is not guaranteed to be monotonically increasing. Don't use it for ordering. Use `timestamp`.

    In practice it often is, but you should not rely on it because we don't take any measures to ensure that and it might change in the future without warning.

2) Reverse the order. The oldest message should be returned first.